chapters/annotations.tex

```diff
@@ -924,7 +924,9 @@ \subsubsection{Graphical Properties}\label{graphical-properties}
 The \lstinline!FillPattern! attributes \lstinline!Horizontal!, \lstinline!Vertical!, \lstinline!Cross!, \lstinline!Forward!, \lstinline!Backward! and \lstinline!CrossDiag! specify fill patterns drawn with the line color over the fill color.
 
 The attributes \lstinline!HorizontalCylinder!, \lstinline!VerticalCylinder! and \lstinline!Sphere! specify gradients that represent a horizontal cylinder, a vertical cylinder and a sphere, respectively.
-The gradient goes from line color to fill color.
+The gradients colors are defined considering full rectangles (ignoring rounded corners) and full ellipses (ignoring arcs).
+For polygons the gradients are defined on the enclosing rectangle (for \lstinline!HorizontalCylinder!, \lstinline!VerticalCylinder!) or ellipse (for \lstinline!Sphere!).
+The color gradient attains the fill color at the center, and attains the line color at two of the sides of the rectangle or the border of the ellipse.
```

Comment:
To clearly rule out one of the gradients we have seen in images posted in this PR, this needs to be more clear for the cylinders:

```suggestion
For \lstinline!HorizontalCylinder!, the gradient attains the fill color along the horizontal line dividing the enclosing rectangle at the center, and the line color at the two horizontal sides of the rectangle.
The \lstinline!VerticalCylinder! is defined analogously to \lstinline!HorizontalCylinder!, but using vertical lines.
For \lstinline!Sphere!, the gradient attains the fill color at the center of the enclosing ellipse, and the line color along the entire border of the ellipse.
```